content/metadata.yaml

```diff
@@ -320,12 +320,13 @@ author_info:
   #   twitter: null
   #   affiliations:
   #   - Flatiron Institute
-  # - name: Casey Greene
-  #   orcid: null
-  #   github: null
-  #   twitter: null
-  #   affiliations:
-  #   - University of Pennsylvania
+   - name: Casey Greene
```

Comment:
Build may be failing because of an extra space in each of these indentations, i.e. 3 instead of 2 and 5 instead of 4.

@dhimmel the strict yaml formatting is going to be tricky for the workshop.  I didn't check your slides, but it may be helpful to be precise about the instructions for editing `metadata.yaml`.

Reply:
Okay.